### ISSUE TITLE
Students shouldn't see changes made by teacher in replay mode.

### ISSUE THREAD

**Issue by b20thelo** [HGustavs/LenaSYS#12145]
When students access a diagram variant created by a teacher, it shouldn't be possible to use the replay mode feature to see how the teacher created the diagram provided. 

Replay mode accessable here: 
<img src="https://user-images.githubusercontent.com/81772705/166813676-ebd09c35-3c13-4408-89ca-414e202e3f58.png" height="200">

When the student clicks the play button in the image below or uses the mouse to pull the slider back and forth to view the changes (Henrik calls it scrubbing), the start position for the replay mode should be what happened after the student loaded the dugga for the first time. 
<img src="https://user-images.githubusercontent.com/81772705/166813528-88044ba1-12af-479f-aba2-754170e3d526.png" height="200">

In other words, the replay feature shown ONLY show changes made by the student.

**Comment by b20matad** [comment]
After several hours of research and coding we found that the functionality already existed.
The "Save-button" will download a .JSON file of the diagram, with history.
The "Export-button" will download a .JSON file of the diagram, without history.

To make this clearer our suggestion is to change the name of the buttons like this:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/102598238/167632268-b6c7e17b-95cf-4e92-9af1-a128c37b773a.png">
